### PR TITLE
Bg animation

### PR DESCRIPTION
**Pull request by soerface**
Stars are rotating and twinkling, and every minute, a new rocket flies across the screen.
Maybe twinkling layer is too slow (didn't want to increase sprite size, so I can't move it that long into one direction without making it disappear).

@nikeee Estimated render time on my server is about 2 weeks for 3 hours of 1080p 60fps video (3 days for 480p 15fps). Do you have a stronger server at hand?

### CODE REVIEW

**Comment by soerface** [comment]
2 Previews (2 minutes stars + rocket, 10 minutes only rockets)

[Background-Background-620613.zip](https://github.com/flipdot/0xA-video-intro/files/3721406/Background-Background-620613.zip)

File: render.py [hunk 1]
```diff
@@ -12,33 +12,36 @@
...
-    Talk('dargmuesli', 'DargStack: Ein Strauß Microservices', None),
-    Talk('Antares', 'Blast Procedure – Wie man Party und Videogames vereinen kann', None),
-    Talk('Sven', 'Esperanto: Wie funktioniert eine Plansprache?', None),
-    Talk('nikeee', 'Wie zählt man bei \$BIG\_SITE die Seitenaufrufe?', None),
-    Talk('analogmultiplizierer', 'flipdot Badge PCB Design', None),
+    Talk('dargmuesli', 'DargStack: Ein Strauß Microservices', '187721'),
+    Talk('Antares', 'Blast Procedure - Wie man Party und Videogames vereinen kann', '449339'),
+    Talk('Sven', 'Esperanto: Wie funktioniert eine Plansprache?', '563460'),
+    Talk('nikeee', 'Wie zählt man bei \$BIG\_SITE die Seitenaufrufe?', '519098'),
+    Talk('analogmultiplizierer', 'flipdot Badge PCB Design', '600089'),
```
Comment (soerface) [inline render.py:31]: Those changes already exist in master, I just pushed master too late. Dunno if GH will update automatically.

**Comment by nikeee** [comment]
I think one minute is too frequent. Maybe 5 minutes is appropriate.

I have a machine that has an ETR of `5.13` days for the 3 hour 1080p 60fps version.

**Comment by soerface** [comment]
5 minutes sounds good to me. Would you start rendering with the new setting? Thanks.

**Pull request merged**